reference/entry.json

```diff
@@ -670,6 +670,108 @@
         ]
       }
     },
+    "/events/{eventId}/videos": {
+      "parameters": [
+        {
+          "$ref": "#/components/parameters/eventId"
+        }
+      ],
+      "post": {
+        "summary": "Add a video to an event",
+        "operationId": "post-event-video",
+        "responses": {
+          "200": {
```

Comment:
This should probably be a `204` (No Content)?

Reply:
+ Same for the place variant

Reply:
Because adding a video to an event or place gives it a uuid that is also used inside the projection I preferred returning a `videoId` here.

Reply:
Okay I glanced over that in the docs then. Can you add an example response? (Also for the place operation)

Reply:
Added an example for place and event